libgnucash/app-utils/calculation/expression_parser.c

```diff
@@ -165,7 +165,7 @@
  * later, when the parser attempts to free the memory through a call
  * to "free_numeric". In addition, freeing the memory will probably
  * invalidate the numeric value contained therein and lead to
- * pernicuous results when the value is used.
+ * pernicuous [??] results when the value is used.
```

Comment:
I'm not sure what's intended. (The obvious word doesn't make particular sense in context.)

Reply:
"pernicious"

Reply:
https://github.com/Gnucash/gnucash/commit/e2fd770528825fd9adf1fe086cb68a8bc2ccacd0